### ISSUE TITLE
How to realize the sg_conv() in pytorch?

### ISSUE THREAD

**Issue by pipisjy** [buriburisuri/sugartensor#46]
v2=v1.sg_conv(dim=64, size=(1,1),  name='gen1',pad="SAME",bn=True)

conv2 = torch.nn.conv2d(16,64,kernel_size=(1,1), bias=True, padding=0)
bn2 = nn.BatchNorm2d(64)
bn2(conv2(v1)

are these two equal ? 
thank you
